src/UI/MainMenuOptions.h

```diff
@@ -8,11 +8,11 @@ class MainMenuOptions : public Window
     MainMenuOptions();
     virtual ~MainMenuOptions();
 
-    virtual void update();
+    virtual void update() override;
 
 protected:
     virtual void init();
-    void visibilityChanged(bool visible) override;
+    virtual void visibilityChanged(bool visible) override;
```

Comment:
The `virtual` can be omitted when using `override`. Not sure if anyone wants to comment about preference, though I think most existing code that uses `override` omits `virtual`. A quick grep shows we've been a little inconsistent about it.

Reply:
I was trying to at least be consistent within the file.

Reply:
Ahh, I see. The diff display made that a little less obvious looking.

I suppose we can look into dropping `virtual` in a different PR. Similarly we can look into being more consistent about `override` in a different PR.